src/plone/restapi/serializer/dxfields.py

```diff
@@ -29,6 +36,20 @@ def get_value(self, default=None):
                        default)
 
 
+@adapter(IRichText, Schema, Interface)
+@implementer(IFieldSerializer)
+class RichTextFieldSerializer(DefaultFieldSerializer):
```

Comment:
Is this serializer really needed? Shouldn't the `SchemaFieldSerializer` below also work perfectly for `IRichText`?

Aside from that, somehow I don't like to have this serializer with this name in this module because it may be confusing and would lead to naming conflicts if we would need to implement a `RichTextFieldSerializer` for `IDexterityContent`.